composer.json

```diff
@@ -3,18 +3,18 @@
     "description": "A package of useful components for internationalisation in Laravel applications.",
```

Comment:
Could we make this package private?

As this package will need to be updated on all its consumers, could we rename the repo too and get rid of `pod-point/countries` once and for all for `pod-point/i18n` instead?

These might be OOS here, if so, happy for additional cards to be created.